webapp/src/components/FileList.vue

```diff
@@ -93,6 +101,26 @@ export default {
   },
   methods: {
     formatDistance,
+    getFileSize(file) {
+    return (
+      file.size ||
+      file.size_bytes ||
+      file.file_size ||
+      file.filesize ||
+      file.length ||
+      file.bytes ||
+      file.meta?.size ||
+      file.file_information?.size ||
+      file.file_information?.size_bytes ||
+      null
+    );
+  },
+    formatFileSize(size) {
+    if (size < 1024) return `${size} B`;
+    if (size < 1024 * 1024) return `${(size / 1024).toFixed(1)} KB`;
+    if (size < 1024 * 1024 * 1024) return `${(size / 1024 / 1024).toFixed(1)} MB`;
+    return `${(size / 1024 / 1024 / 1024).toFixed(2)} GB`;
```

Comment:
```suggestion
    formatFileSize(size_bytes) {
    if (size_bytes < 1024) return `${size_bytes} B`;
    if (size_bytes < 1024 * 1024) return `${(size_bytes / 1024).toFixed(1)} KB`;
    if (size_bytes < 1024 * 1024 * 1024) return `${(size_bytes / 1024 / 1024).toFixed(1)} MB`;
    return `${(size_bytes / 1024 / 1024 / 1024).toFixed(2)} GB`;
```

I think we can use a slightly more descriptive variable name here to get away without a docstring/comment